extensions/resource-deployment/src/services/tools/azCliTool.ts

```diff
@@ -3,21 +3,24 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ToolType } from '../../interfaces';
-import * as nls from 'vscode-nls';
+import { EOL } from 'os';
 import { SemVer } from 'semver';
+import * as nls from 'vscode-nls';
+import { Command, OsType, ToolType } from '../../interfaces';
 import { IPlatformService } from '../platformService';
-import { EOL } from 'os';
 import { ToolBase } from './toolBase';
+
 const localize = nls.loadMessageBundle();
+const defaultInstallationRoot = '~/.local/bin';
+const win32InstallationRoot = `${process.env['ProgramFiles(x86)']}\\Microsoft SDKs\\Azure\\CLI2\\wbin`;
```

Comment:
Ensure this is the best practice. The user will have to open this application as administrator for that to work.

Reply:
We do not have a choice on the location of this install so far as I understand. The Azure CLI that windows azure ships always installs at that location.